rmf_demo_panel/rmf_demo_panel/static/src/components/tasks/task-card.tsx

```diff
@@ -72,7 +77,8 @@ export const TaskCard = (props: TaskCardProps) : React.ReactElement => {
                     </Typography>
                   </Grid>
                   <Grid item xs={6}>
-                    <Chip color={returnStateColor(taskState.state)} label={taskState.state} size='small'/>
+                    { !isDelayed && <StyledChip state={taskState.state}/> }
+                    { isDelayed && <StyledChip state="Delayed"/> }
```

Comment:
One thing to comment. It will be good if we can retain the task state as taskState.state. 
Instead of showing "Delayed" here, would prefer to show the "Delayed" on the progress bar/meter, substitute the percent with "Delayed".